src/constants.js

```diff
@@ -1,40 +1,99 @@
-export const NETWORK_TYPE = {
-  ethereum: 'ethereum'
-};
+import _ from 'lodash';
+import colors from './colors';
 
-export const NETWORK_IDS = {
-  '0': 'olympic',
-  '1': 'frontier',
-  '2': 'expanse',
-  '3': 'ropsten',
-  '4': 'rinkeby',
-  '5': 'goerli',
-  '8': 'ubiq',
-  '42': 'kovan',
-  '61': 'classic',
-  '77': 'sokol',
-  '99': 'core',
-  '7762959': 'musicoin'
-};
+export const NetworkTypes = Object.freeze({
+  ETHEREUM: 'ethereum',
+  SUBSTRATE: 'substrate'
+});
 
-export const NETWORK_ID = Object.entries(NETWORK_IDS).reduce(
-  (acc, [key, value]) => Object.assign(acc, { [value]: key }),
-  {}
-);
+export const EthereumNetworkIds = Object.freeze({
+  OLYMPIC: '0',
+  FRONTIER: '1',
+  EXPANSE: '2',
+  ROPSTEN: '3',
+  RINKEBY: '4',
+  GOERLI: '5',
+  UBIG: '8',
+  KOVAN: '42',
+  CLASSIC: '61',
+  SOKOL: '77',
+  CORE: '99',
+  MUSICOIN: '7762959'
+});
 
-export const NETWORK_LIST = ['1', '61', '3', '42', '5'];
+export const SubstrateNetworkIds = Object.freeze({
+  KUSAMA: 's0'
+});
 
-export const NETWORK_TITLES = {
-  [NETWORK_ID.frontier]: 'Ethereum',
-  [NETWORK_ID.classic]: 'Ethereum Classic',
-  [NETWORK_ID.ropsten]: 'Ropsten Testnet',
-  [NETWORK_ID.kovan]: 'Kovan Testnet',
-  [NETWORK_ID.goerli]: 'Görli Testnet'
+const substrateNetworkRaw = {
+  [SubstrateNetworkIds.KUSAMA]: {
+    name: 'kusama',
+    ss58Prefix: 2,
+    balanceModuleId: 123
```

Comment:
question: is this the actual balanceModuleId for Kusama? if not, could you please leave a comment in case someone else is trying to figure that out in the future :)

Reply:
No, I do not know what the balanceModuleId is, I will leave the comment